test/bats/configs/nginx/templates/nginx.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   serviceAccountName: nginx-{{ .Values.sa }}
   terminationGracePeriodSeconds: 0
   containers:
-  - image: docker.mirror.hashicorp.services/nginx
+  - image: docker.mirror.hashicorp.services/nginxinc/nginx-unprivileged
```

Comment:
The unprivileged nginx container is a lot easier to work with on OpenShift, and we don't need a privileged version for the tests, so I switched everything to the unprivileged version.